src/test/java/com/internship/backend/config/UsernamePasswordAuthenticationProviderTest.java

```diff
@@ -44,10 +43,10 @@ void authenticate_ValidCredentials() {
 
         User user = new User();
```

Comment:
could you extract this part in a separate method and use it in the rest of the tests?